koku/api/user_access/view.py

```diff
@@ -28,121 +28,88 @@ class UIFeatureAccess:
         1. to keep UI and API in sync about RBAC permissions
         2. to provide the UI (both our UI and the platform's chroming) with a simple boolean response regarding whether
             a user has access to specific features of the UI.
-
-    Class attributes:
-
-        access_keys (list) - a list of access keys to check. must be implemented by sub-classes.
-                             multiple keys will be ORed; i.e. if a user has access to anything in the list,
-                             this API returns True
     """
 
-    def __init__(self, access):
+    PRERELEASE = ["ibm"]
+
+    def __init__(self, access, admin_user, access_key_list=None):
         """Class Constructor.
 
         Args:
             access (dict) - an RBAC dict; see: koku.koku.middleware.IdentityHeaderMiddleware
 
         """
+        if not access_key_list:
+            self.access_key_list = self.access_key_mapping.keys()
         self.access_dict = access if access else {}
+        self.admin_user = admin_user
+        self.access_key_list = []
```

Comment:
I'm a little confused here. I can't seem to find where we pass in `access_key_list`, so the first `if not access_key_list` is always true, then we set `self.access_key_list = self.access_key_mapping.keys()`. But then right here we do `self.access_key_list = []`. We set, then immediately unset this value.